spacepy/pybats/rim.py

```diff
@@ -387,7 +374,7 @@ def add_cont(self, var, target=None, n=50, maxz=False, lines=False,
             that is strictly positive and "Seismic" for diverging data.
             Alternatively, legacy Ridley Ionosphere Model color maps can be
             loaded using "l_wr" (white red) or "l_bwr" (blue-white-red),
-            where the "l_" prefix indicates legacy and not Matplotlib color maps.
+            where the "l" prefix indicates legacy and not Matplotlib color maps.
```

Comment:
Again, the underscore probably matters. Please check through for this pattern so I don't have to highlight them all.

Reply:
resolved